config.yaml

```
@@ -130,3 +130,21 @@ options:
       The current release deployed is available by viewing
         juju status azure-cloud-provider
 
+  azuredisk-release:
+    type: string
+    description: |
+      Specify the version of csi-azuredisk as defined by the `releases`
+      directory of https://github.com/kubernetes-sigs/azuredisk-csi-driver/
+
+      example)
+        juju config azuredisk-release provider-release='v1.24.0'
+      
+      A list of supported versions is available through the action:
+        juju run-action azure-cloud-provider/0 list-releases --wait
+      
+      To reset by to the latest supported by the charm use:
```

```suggestion
        juju config azure-cloud-provider azuredisk-release='v1.21.0'
      
      A list of supported versions is available through the action:
        juju run-action azure-cloud-provider/0 list-releases --wait
      
      Reset to the default release supported by the charm with:
```

i didn't see a `v1.24.0` release in the [azuredisk repo](https://github.com/kubernetes-sigs/azuredisk-csi-driver/releases) so i set the default to the latest one i saw (1.21.0).